CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Native function calling support to `OpenAiChatPromptDriver`, `AzureOpenAiChatPromptDriver`, `AnthropicPromptDriver`, `AmazonBedrockPromptDriver`, `GooglePromptDriver`, and `CoherePromptDriver`.
 - `OllamaEmbeddingDriver` for generating embeddings with Ollama.
 - `GriptapeCloudKnowledgeBaseVectorStoreDriver` to query Griptape Cloud Knowledge Bases.
+- `GenericArtifact` for storing any data.
```

Comment:
I'm wondering if `GriptapeArtifact` or event just `ArtifactWrapper` make sense here? Barring those options, I'm kind of more for just `Artifact`. 

Essentially I see this as "make the random information of mine into an object type that Griptape understands", so while `GenericArtifact` gets the point across, it's not very descriptive for what the intent is.

Reply:
@emjay07 take a look at the updated example, I simplified the syntax quite a bit.